server/config_test.go

```diff
@@ -15,6 +15,8 @@ func TestLoadUsers(t *testing.T) {
 		expUsers []user.UserWithRemoteIdentities
 		// userid -> plaintext password
 		expPasswds map[string]string
+
+		wantErr bool
```

Comment:
perhaps change to `expectedErr error` and compare to nil instead, or a specific error?